lstchain/scripts/lstchain_mc_rfperformance.py

```diff
@@ -111,80 +111,33 @@ def main():
 
     ####PLOT SOME RESULTS#####
 
-    gammas = dl2[dl2.gammaness >= 0.5]
-    protons = dl2[dl2.gammaness < 0.5]
-    gammas.reco_type = 0
-    protons.reco_type = 1
-
-    focal_length = 28 * u.m
-    src_pos_reco = utils.reco_source_position_sky(gammas.x.values * u.m,
-                                                  gammas.y.values * u.m,
-                                                  gammas.reco_disp_dx.values * u.m,
-                                                  gammas.reco_disp_dy.values * u.m,
-                                                  focal_length,
-                                                  gammas.mc_alt_tel.values * u.rad,
-                                                  gammas.mc_az_tel.values * u.rad)
-
+    selected_gammas = dl2.query('reco_type==0 & mc_type==0')
```

Comment:
does this imply you are selecting only events that have `reco_type == 0`?

Reply:
Yes, this is equivalent to the previous `dl2.gammaness >= 0.5` but ensuring consistent in the threshold applied in the reconstruction

Reply:
I forgot to ask... I guess that within this query we can request different gammaness thresholds, e.g. `dl2.query('dl2.gammaness >= 0.9')`? or how would that call be in this case?

Reply:
That would be just `'gammaness >= 0.9'` but yes sure. It works like masks.